### ISSUE TITLE
prune policy

### ISSUE THREAD

**Issue by ThomasWaldmann** [marcpope/bbs#2]
seen daily, weekly, monthly on your screenshot.
maybe add hourly and yearly (at least).

**Comment by marcpope** [comment]
Good catch, I will add yearly. The fields change depending on the policy. If you are backing up every hour, then hourly shows. If you are backing up minutely then minute pruning also shows. There would be no point to prune minutely for a backup happening every hour. I am not going to mess with secondly because the cron that checks for tasks will only will run every minute.

I am torn on how to do pruning. In the documentation it says it's not as efficient to do it after every backup, but in the quick start, that's essentially what it is doing. From a programming standpoint, it would be easier to do pruning after each backup.

**Comment by marcpope** [comment]
Ok. I'll try to break those tasks apart
